### PR TITLE
Drack dev

### PR DESCRIPTION
**Pull request by Drackion**
The initial gregified ae2 recipes
most of the base stuff is gregified

### CODE REVIEW

File: kubejs/server_scripts/mods/ae2/gregifyAE2.js [hunk 1]
```diff
@@ -0,0 +1,1897 @@
...
+            .itemInputs(
+                'gtceu:certus_quartz_gem',
+                'ae2:fluix_dust',
+                'ae2:logic_processor'
+            )
+            .inputFluids(
+                Fluid.of(
+                    'gtceu:red_alloy',
+                    144
+                )
```
Comment (screret) [inline kubejs/server_scripts/mods/ae2/gregifyAE2.js:24]: could you just do `'gtceu:red_alloy 144'`? it'd be way more compact
or at least put this on 1 line

File: kubejs/server_scripts/mods/ae2/gregifyAE2.js [hunk 2]
```diff
@@ -0,0 +1,1897 @@
...
+            )
+            .itemOutputs(
+                '2x ae2:formation_core'
+            )
```
Comment (screret) [inline kubejs/server_scripts/mods/ae2/gregifyAE2.js:28]: here, just make the 1 thing i/o's a single line, it's more readable that way

**Review by screret — CHANGES_REQUESTED** [review]

**Comment by screret** [comment]
apply the requested changes to every instance of the thing btw.

**Comment by Drackion** [comment]
I'll commit those changes tomorrow as i am off my PC fire the night

**Comment by thevortex** [comment]
i can make the changes if you like @Drackion

**Comment by Drackion** [comment]
@thevortex sure thx

**Pull request merged**